### ISSUE TITLE
request for comments on newsletter process, team and compensation

### ISSUE THREAD

**Issue by NiklasTR** [labdao/newsletter#2]

**Comment by jocelynnpearl** [comment]
Solid first issue and team effort. I put together the headline/banner graphic incorporating the new logo. The community WG collected the items for the both the newsletter and community announcement (discord) during our weekly wg call. Hossam, Alex Dobrin, Jan and Niklas all helped with the Revue side of things. Editing input from Boris. I thought it went pretty smoothly. I also scheduled a couple promo tweets including a sign up/ link in bio type graphic - would like feedback on number of subscribers. We have not done a major call to action in the community yet to sign up. Overall we're getting there! Now we just need a process for regular scheduled newsletter (what is the cadence? once a month?)

**Comment by NiklasTR** [comment]
I suggest once a month - I put together a README in this repository with comments on the process - would love to hear your thoughts and potential changes

**Comment by tylerstahl** [comment]
Yeah, we discussed a little about this at the community call the other day. Not sure if you saw the notes but to recap: 
- Monthly newsletter (unless big/important announcement) 
- 5 minute recorded “conversation” / news report alternative to the newsletter 

Additionally, I will add:
- We need a place to collaborate on the Newsletter (I didn't contribute to this one so maybe this exists), and maybe integrate this within dework as well 
- I also volunteered to start taking some big picture notes at the tech meetings to include in the newsletter and share internally 

As far as compensation, I think if we structure it similar to VitaDAO? Which I don't know the exact details

**Comment by bjdyakov** [comment]
I want to echo what Jocelynn said: great work on the first issue!!

@NiklasTR I think the process you proposed sounds solid. An idea I have is to somehow integrate all of this with a **weekly** update info collection process. The idea being that each we we share an internal weekly recap on discord (in the new #week-in-review channel I set up today), and then draw on those on a monthly basis to produce a newsletter which covers all the key updates for folks who aren't checking discord .

On that note, @NiklasTR @hossam-zaki please let me know what tech WG progress you'd like to give updates on for this week so I can post the weekly recap tomorrow :)

**Comment by hossam-zaki** [comment]
I thought that the first issue went great. I also think that we should have the newsletter be monthly so that we have enough information to put into the newsletter itself. Having a weekly review channel i think is great. 

HackMD would be great to use for writing the newsletter. My one qualm is that we might not have the grammar checking functionality that we would have on google docs.

Can we integrate hackmd into github? Ie, once a newsletter is done, we push it to the repo.

**Comment by tylerstahl** [comment]
Note for self, integrate process for DeWork into Newsletter
